### DISCUSSION TITLE
posts/java08/

### DISCUSSION THREAD

**Discussion by giscus** [hyunminkim93/hyunminkim93.github.io#3] [Announcements]
# posts/java08/

레이아웃 이미지 슬라이드에 관하여

https://hyunminkim93.github.io/posts/java08/

**Comment by an83690880** [comment]
알기 쉽게 적어 놓으셨네요!!  재밌게 보고 가요~~!!

**Comment by kimyih** [comment]
잘보고갑니다 !! 감사합니다
